src/components/Orderbook.tsx

```diff
@@ -102,6 +103,10 @@ interface OrderTableEntry {
   bondValue: {
     value: number
     displayValue: string // example: "0" (no fb) or "114557102085.28133"
+    locktime?: number
```

Comment:
This part looks fine! Should amount also be explicitly converted to string?

Reply:
Hmm.. I think I know what you mean.. `amount` is passed as `String` to the `Balance` component. I think I'd rather want to have the type info here in the object rather than storing it as plain string here. Okay for you?